packages/dnd/src/drop-cursor/drop-cursor.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { createRef, Fragment, useCallback, useEffect, useRef } from "react";
+
+import findTargetElement from "../find-target-element";
+import useDispatch from "../use-dispatch";
+import useState from "../use-state";
+
+const DropCursor: React.FC<{ CursorProps?: any }> = ({ CursorProps }) => {
```

Comment:
You might not want to use the `React.FC` or `React.ComponentType`. It automatically allows the `children` prop.